config/boot.rb

```diff
@@ -1,4 +1,4 @@
-ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)
+ApplicationConfig["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)
```

Comment:
I'm not sure if `ApplicationConfig` should be used in `config` files as well, but I assume it should be.

Reply:
Specifically `ApplicationConfig` can be used anywhere after it's been initialized.

Getting this error... `home/travis/build/thepracticaldev/dev.to/config/boot.rb:1:in `<top (required)>': uninitialized constant ApplicationConfig (NameError) ` in the build. So there's your answer 😄

`ApplicationConfig` is designed to let us ensure the right config vars are in place before booting, and the defaults get set if they're not. So no need it to use it in contexts where that doesn't matter. So probably good that it was included in a couple new files and initializers, but when it's included before the initializers it breaks the build.